### ISSUE TITLE
Technique C7 in WCAG 21 is missing the css code example

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#566]
The copy of this technique is missing the CSS example. 
```a span { height: 1px; width: 1px; position: absolute; overflow: hidden; top: -10px; }```
https://www.w3.org/WAI/WCAG21/Techniques/css/C7   WCAG 2.1

Here's the original 
https://www.w3.org/TR/WCAG20-TECHS/C7.html    WCAG 2.0

**Comment by alastc** [comment]
Good spot @DavidMacDonald, I think it's just missing tags around that bit of code so is ignored by the build process. I just put in a PR for Michael to review.

**Comment by michael-n-cooper** [comment]
Fixed by #586
